bin/utils.sh

```diff
@@ -59,14 +59,16 @@ function common_shutdown {
 }
 
 #---
-# args:               Process name, coordinator or shuffle-server
+# args:               Process name, coordinator, shuffle-server or uniffle-dashboard
 #---
 function get_pid_file_name {
   process_name="$1"
   if [[ $process_name == "coordinator" ]]; then
     pid_file="coordinator.pid"
   elif [[ $process_name == "shuffle-server" ]]; then
     pid_file="shuffle-server.pid"
+  elif [[ $process_name == "uniffle-dashboard" ]]; then
```

Comment:
It's called dashboard.Too long.

Reply:
OK, I've already renamed it to "dashboard".

Reply:
LGTM, thanks